optimizely/project_config.py

```diff
@@ -557,3 +566,35 @@ def is_feature_experiment(self, experiment_id):
         """
 
         return experiment_id in self.experiment_feature_map
+
+    def get_variation_from_id_by_experiment_id(self, experiment_id, variation_id):
+        """ Gets experiment id and variation id
+
+            Returns:
+                The variation for the experiment id and variation id
+                or empty dict if not found
+        """
+        if (experiment_id in self.variation_id_map_by_experiment_id and
+                variation_id in self.variation_id_map_by_experiment_id[experiment_id]):
+            return self.variation_id_map_by_experiment_id[experiment_id][variation_id]
+
+        self.logger.error('Variation with id "%s" not defined in the datafile for experiment "%s".',
+                          variation_id, experiment_id)
+
+        return {}
+
+    def get_variation_from_key_by_experiment_id(self, experiment_id, variation_key):
+        """ Gets experiment id and variation key
```

Comment:
please rephrase and have more clear explanation.

Reply:
comments changed to better describe what is happening.